package.json

```diff
@@ -70,7 +70,7 @@
     "circleci": "NODE_ENV=circleci npm run hint && mocha --reporter mocha-circleci-reporter --reporter-options mochaFile=$CIRCLE_TEST_REPORTS/junit/junit.xml test",
     "start": "npm run hint && node .",
     "dev": "nodemon -e js,hbs -w app",
-    "hint": "eslint '**/*.js' --quiet",
+    "hint": "eslint 'app/**/*.js' --quiet",
```

Comment:
ah yes, we should rename this to `server`, see the guideline on https://github.com/OpenCollective/OpenCollective/tree/master/dev/default_folder_structure

(now is a good time to harmonize all of this across repos)